### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by TheMezStrikes**
Emotes now leave timestamps and even work in PM!!

### CODE REVIEW

**Comment by TheMezStrikes** [comment]
Like the idiot I am, I made a little coding mistake that is fixed now

**Comment by TheMezStrikes** [comment]
Userlist order will now always be correct, following both the standard sorting by rank level (leadership, staff, normal, punishment) but staying true to the order provided in config.js.

**Comment by TheMezStrikes** [comment]
~~I was just wondering what that button did~~

File: mergesort.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+(function() {
+	
+var MSort = function(array, comparator){
```
Comment (Spandamn) [inline mergesort.js]: Can't you just use `Array.sort()` Instead of creating a new file?

**Pull request merged**